### PR TITLE
More cleanups

### PR DESCRIPTION
**Pull request by pvuorela**

### CODE REVIEW

**Comment by pvuorela** [comment]
@rainemak

File: plugin/ZoomableThumbnail.qml [hunk 1]
```diff
@@ -24,6 +24,8 @@ SilicaFlickable {
     id: base
 
     property real maxHeight: height
+    property real minWidth: base.width
+    property real maxWidth: base.width * 2.5
```
Comment (rainemak) [inline plugin/ZoomableThumbnail.qml]: These two look like readonly properties.

File: main.cpp [hunk 2]
```diff
@@ -18,12 +18,70 @@
...
+
+    QTranslator *engineeringEnglish = new QTranslator(app.data());
+    if (!engineeringEnglish->load("sailfish-office_eng_en", TRANSLATION_INSTALL_DIR))
+        qWarning("Could not load engineering english translation file!");
+    QCoreApplication::installTranslator(engineeringEnglish);
+
+    QTranslator *translator = new QTranslator(app.data());
+    if (!translator->load(QLocale::system(), "sailfish-office", "-", TRANSLATION_INSTALL_DIR))
+        qWarning() << "Could not load translations for" << QLocale::system().name().toLatin1();
+    QCoreApplication::installTranslator( translator );
```
Comment (rainemak) [inline main.cpp]: Extra spaces

**Comment by rainemak** [comment]
@pvuorela added two comments. Please give a look.

**Comment by pvuorela** [comment]
Changed and included a new commit removing an unused file.

**Comment by rainemak** [comment]
LGTM

**Pull request merged**